src/reducers/__tests__/topics.reducer.test.js

```diff
@@ -0,0 +1,53 @@
+import topicReducer, { initialState } from '../topics.reducer';
+import { topicFetchInit, topicsFetched, topicsFetchError } from '../../actions/topic.actions';
+import TopicModel from '../../shared/models/TopicModel';
+
+describe('topics reducer', () => {
+    
+    it('should have initial state', () => {
+        const topicState = topicReducer(undefined, {});
```

Comment:
hmm... what was the first argument here? I have an idea we could use a variable in such cases that gives this argument a name, although `const myArg = undefined` looks strange

Reply:
if we were talking about any other functions then I would agree. However, the way how `reducer`s are defined by convention it always is `(state, action)`. 
https://redux.js.org/faq/reducers 
``` You just need to make sure that together they follow the basic rules of reducers: (state, action) => newState, and update state immutably rather than mutating it directly.```
I am not convinced that we should make it `const noInitialState = undefined`  - it would only help some newcomers to redux. What do you think?